test/e2e/object-fields-filtering/use-kubestellar.sh

```diff
@@ -18,7 +18,7 @@ set -e # exit on error
 
 :
 : -------------------------------------------------------------------------
-: "Create a bindingpolicy in wds1 to deliver a service to one WEC."
+: "Create a bindingpolicy in wds1 to deliver difficult API objects to one WEC."
```

Comment:
Why can't we just say to deliver Job and Service API objects.  Users may not know what difficult means here

Reply:
reworded.